CMakeLists.txt

```diff
@@ -165,10 +151,10 @@ target_include_directories(QtNodes
 
 target_link_libraries(QtNodes
   PUBLIC
-    ${Qt}::Core
-    ${Qt}::Widgets
-    ${Qt}::Gui
-    ${Qt}::OpenGL
+    Qt${QT_VERSION_MAJOR}::Core
+    Qt${QT_VERSION_MAJOR}::Widgets
+    Qt${QT_VERSION_MAJOR}::Gui
+    Qt${QT_VERSION_MAJOR}::OpenGL
```

Comment:
If we require 5.15 as a minimum version, then we'd be able to write just `Qt::Widgets`, such namespaces are defined in 5.15 and 6.xx